tests/test_utilities.py

```diff
@@ -96,3 +99,18 @@ def test_hss_solver_config_override(self):
             self.assertTrue(FeatureFlags.hss_solver_config_override)
         with isolated_environ(remove=('DWAVE_FEATURE_FLAGS',)):
             self.assertFalse(FeatureFlags.hss_solver_config_override)
+
+
+class TestScheduleWithAnnealOffset(unittest.TestCase):
+    def test_doc_file(self):
+
+        csv_file =  os.path.join(
```

Comment:
For testing, I'd create a simple schedule by hand (few lines), just so that verification is easier.

Reply:
I.e. no need to include the 1k line csv as part of the docs static.

Reply:
OTOH, if you want to test on a realistic schedule, you can include it as a test fixture (see an example [here](https://github.com/dwavesystems/dwave-inspector/tree/master/tests/fixtures/cassettes)).

Reply:
Implemented, thanks!